web/src/App.js

```diff
@@ -6,7 +6,8 @@ import axios from 'axios';
 import { CircularProgress } from '@material-ui/core';
 import urljoin from 'url-join'
 
-const API_ADDRESS = urljoin(window.location.protocol + '//' + window.location.host, 'api')
+// const API_ADDRESS = urljoin(window.location.protocol + '//' + window.location.host, 'api')
+const API_ADDRESS = 'http://localhost:8080/api'
```

Comment:
This shouldn't be committed right? Also, why do we need to change the API address now that we have skaffold for local testing?

Reply:
My bad, I will remove the test URL from the commit. I still use a local test server to see my changes in real time as the `npm build` and `skaffold` can take a long time.